packages/boot/package.json

```diff
@@ -19,9 +18,17 @@
   "author": "Agoric",
   "license": "Apache-2.0",
   "dependencies": {
+    "@agoric/assert": "^0.6.0",
```

Comment:
Why do these need promotion to full dependencies, given that the benchmark (`bench-vaults-performance`) was moved *out* of this package? I kind of expect they'd be dependencies of the new `packages/benchmark`, but not this one.

Reply:
Alas, a bunch of stuff that I wanted to move into `packages/benchmark` (and which were so relocated in the original version of this PR) are just moved to the `tools` directory of this package.  There was some controversy about where various things belong, and the easiest immediate resolution was to leave them here.  I would like to revisit this in a future PR, but I don't want to hold up landing this functionality for the resolution of this matter.